test/journeys/specs/oneOnOne/messaging.js

```diff
@@ -114,13 +122,94 @@ describe(`Widget Space: One on One`, () => {
 
     it(`sends and deletes message`);
 
-    it(`sends message with pdf attachment`);
+    describe(`File Transfer Tests`, () => {
+      /* eslint no-sync: "off" */
+      const uploadDir = path.join(__dirname, `assets`);
+      const downloadDir = path.join(os.homedir(), `Downloads`);
+      const inputFileElem = `.ciscospark-file-input`;
+      const downloadFileButton = `(//div[@title="Download this file"]/parent::button)[last()]`;
```

Comment:
No, I've created a ticket and will make sure to update the tests as well once it is done. https://voxeolabs.atlassian.net/browse/SSDK-1051